doc/source/examples/dem/dem.rst

```diff
@@ -63,7 +64,7 @@ We organize the DEM examples from the simplest to the most complicated example:
 
       dem_14 [label="Granular Mixer", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/granular-mixer/granular-mixer.html"];
 
-      
+      dem_15 [label="Granuheap", href="https://chaos-polymtl.github.io/lethe/documentation/examples/dem/granuheap/granuheap.html"];
```

Comment:
I think my PR (another DEM example) will be merged before this one. So I just let you know that it is normal if you have conflit here when rebasing :)

Reply:
I just fixed it :)